Added instructions for the Microsoft Graph

tvuylsteke opened the pull request:
I noticed quite a bit of people are struggling (which is visible in the comments section of this article) with getting the properties to show up in the Microsoft Graph so I added some detailed instructions.

azurecla commented:
Hi __@devinrader__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

deneha commented:
@devinrader Thanks for fixing the image!

tvuylsteke committed:
Added instructions for the Microsoft Graph

I noticed quite a bit of people are struggling (which is visible in the comments section of this article) with getting the properties to show up in the Microsoft Graph so I added some detailed instructions.

PatAltimore commented on articles/active-directory/connect/active-directory-aadconnectsync-feature-directory-extensions.md:
```diff
@@ -41,9 +41,21 @@ An object in Azure AD can have up to 100 directory extensions attributes. The ma
+
+We can query the Microsoft Graph API through the Microsoft Graph explorer: [https://developer.microsoft.com/en-us/graph/graph-explorer#](https://developer.microsoft.com/en-us/graph/graph-explorer#)
+
+There are some considerations when working with the Microsoft Graph explorer:
```
From this point on, this is probably too much detail since this isn't an article for MS Graph or MS Graph Explorer.  Ideally, there would be a document on how to use graph explorer.  I don't there is one.

PatAltimore replied:
@tvuylsteke  Thanks for the contribution.  I'd suggest dropping the rest of the detail.

tvuylsteke replied:
I get the point that the article is not on graph. Are you suggesting to drop my addition completely? I still think we need it somewhere and it should be linked here. You can clearly see in the comment section that people are really not finding this info. I've seen customers struggle with this as well. We could drop the info on the AAD Graph and only talk about the Microsoft Graph. The Microsoft Graph is the preferred one I think. @AndKjell Any thoughts from your side?

AndKjell replied:
Can we point to https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-graph-api-quickstart#using-the-graph-explorer? I needed, we can add additional information on the Azure AD Graph page.

tvuylsteke replied:
That's a reference to the "old" Azure AD Graph. At the top of that article you have the "we recommend the Microsoft Graph" statement. The Microsoft Graph is documented here: https://developer.microsoft.com/en-us/graph/docs/concepts/overview The docs aren't bad, but you can clearly see that there's a delta with the audience here. "Infra" people get in touch with AAD Connect and all run against the same wall. The doc is way to dev/API oriented.

Still. I get your POV. I guess it's a bad practice documenting how other items work. 

If you agree I'll trim down on the details and just point to this link: https://developer.microsoft.com/en-us/graph/docs/concepts/query_parameters#select-parameter That should get people going as a hint on how to get their property. And I can leave out the clue on how to login/raise permissions as that can be considered basic Microsoft Graph Explorer tasks.

tvuylsteke committed:
Reworked Microsoft Graph change

Trimmed down a bit on the details as explaining how to use the Microsoft Graph is outside the scope of this article.

PatAltimore commented on articles/active-directory/connect/active-directory-aadconnectsync-feature-directory-extensions.md:
```diff
@@ -41,22 +41,20 @@ An object in Azure AD can have up to 100 directory extensions attributes. The ma
-
-The attributes are prefixed with extension\_{AppClientId}\_. The AppClientId has the same value for all attributes in your Azure AD tenant.
+>[!NOTE]
+> You need to explicitely ask for the attribute to be returned. This can be done by explicitely selecting the attributes like this: https://graph.microsoft.com/beta/users/abbie.spencer@fabrikamonline.com?$select=extension_9d98ed114c4840d298fad781915f27e4_employeeID,extension_9d98ed114c4840d298fad781915f27e4_division For more information please check [Microsoft Graph: Use query parameters](https:/ ...
```
@tvuylsteke Explicitly is misspelled.

PatAltimore reviewed (commented):
@tvuylsteke I like this a lot better.  Looks good. I just saw one typo.  Could you fix it?  Once updated, I can sign off on the contribution. Thanks!

tvuylsteke committed:
Fixed typo

Explicitly was misspelled

PatAltimore commented on articles/active-directory/connect/active-directory-aadconnectsync-feature-directory-extensions.md:
```diff
@@ -41,10 +41,20 @@ An object in Azure AD can have up to 100 directory extensions attributes. The ma
+We can query the Microsoft Graph API through the Microsoft Graph explorer: [https://developer.microsoft.com/en-us/graph/graph-explorer#](https://developer.microsoft.com/en-us/graph/graph-explorer#)
+
+>[!NOTE]
+> You need to explicitely ask for the attribute to be returned. This can be done by explicitly selecting the attributes like this: https://graph.microsoft.com/beta/users/abbie.spencer@fabrikamonline.com?$select=extension_9d98ed114c4840d298fad781915f27e4_employeeID,extension_9d98ed114c4840d298fad781915f27e4_division For more information please check [Microsoft Graph: Use query parameters](https:// ...
```
@tvuylsteke Unfortunately, you have another instance of "explicitly" misspelled in in the first sentence. Could you fix the typo.  I'll sign-off after the update.

Thanks,
Pat

tvuylsteke replied:
@PatAltimore Done. Terribly sorry! Thanks!

tvuylsteke committed:
Update active-directory-aadconnectsync-feature-directory-extensions.md

fixed another spelling error
